plugins/thumbnails/server/worker.py

```diff
@@ -157,3 +160,53 @@ def attachThumbnail(file, thumbnail, attachToType, attachToId, width, height):
     }
 
     return ModelImporter.model('file').save(thumbnail)
+
+
+def getImage(extension, data):
+    """
+    Check extension of image and opens it.
+
+    :param extension: The extension of the image that needs to be opened.
+    :param data: The image file stream.
+    """
+    if 'dcm' in extension:
```

Comment:
Let's also pass the `file['mimeType']` so it can be used to detect type. Then we can do:

    if  (extension and extension[-1] == 'dcm') or mimeType == 'application/dicom':